runtime/frequency-rococo/src/xcm_config.rs

```diff
@@ -0,0 +1,153 @@
+use super::{
```

Comment:
Similar question, where did these settings come from?

Reply:
Runtime is copied as is for now, have added a chore #294 to refactor

Reply:
rococo runtime is now standalone and I would be include to use --features but will be a task in itself which can be tracked in above chore or converted to story